nx.json

```diff
@@ -1,6 +1,9 @@
 {
   "$schema": "./node_modules/nx/schemas/nx-schema.json",
   "npmScope": "erkjbro-tech",
+  "affected": {
+    "defaultBase": "dev"
```

Comment:
Should look into this a bit more. GitHub Actions should check "affected" files against `main`, but I like to test against `dev` for feature and hotfix branches.